scripts/basic-workflow-tests.sh

```diff
@@ -186,16 +186,16 @@ test_serve() {
     serve_args=()
     if [ "${model_type}" == "base" ]; then
         if [ "${BACKEND}" == "vllm" ]; then
-            model_path="${CACHE_HOME}/instructlab/models/${GRANITE_SAFETENSOR_REPO}"
+            model_path="${CACHE_HOME}/instructlab/models/${GRANITE_SAFETENSOR_REPO}/main/"
```

Comment:
I think we should consider using another variable like: `MODEL_CACHE="${CACHE_HOME}/instructlab/models/"` or something.

Reply:
added, thanks